src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -4046,4 +4064,38 @@ import ageDisplay, { getLowerAgeYears } from '@ncigdc/utils/ageDisplay';
       $.data(this, 'plugin_' + plugin_name, new InCHlib(this, options, extHandlers));
     })
   };
+
+  $.multitask = {
```

Comment:
This is the golden nugget. The code triages and enqueues operations together, in case we ever wanted to handle more tasks in this manner.